chat/public/index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+<meta charset="UTF-8">
+<title></title>
+</head>
+<body>
+ Name:<input type="text" id="Name" />
+ Comment:<input type="text" id="Comment" />
+ <input type="submit" id="button" />
+ <div>
+ <pre id="chatid"/>
+ </div>
+ <script>
+const input =document.getElementById("button");
+input.onclick=()=>{
+  const name=document.getElementById("Name").value;
+  const comment=document.getElementById("Comment").value;
```

Comment:
use different variable names. it will confuse you with your id names in html file.